deploy/vllm/templates/clusterversion.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: apps.kubeblocks.io/v1alpha1
+kind: ClusterVersion
+metadata:
+  name: vllm-baichuan-7b
```

Comment:
Better not to map a model with clusterversion, it seems that the clusterversion is not proper to vLLM, just make the model option available in "values.yaml". It is very weird to define a model in clusterversion.